timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -325,6 +334,32 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         }
     }
 
+    destroy() {
```

Comment:
This method does not get called when the time graph is closed, or when the trace tab is closed, or when the Trace Viewer view is closed.

Reply:
Sorry I was missing https://github.com/theia-ide/theia-trace-extension/pull/342. Now it gets called.